db/app.js

```diff
@@ -9,4 +13,20 @@ app.get("/api", getEndpointsDocumentation);
 
 app.get("/api/topics", getAllTopics);
 
+app.get("/api/articles/:article_id", getArticleById);
+
+app.use((err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ message: "Bad request." });
+  } else next(err);
+});
+
+app.use((err, req, res, next) => {
+  if (err.code === "22003") {
+    res
+      .status(404)
+      .send({ message: "The data with the given id was not found." });
+  } else next(err);
+});
```

Comment:
Since these are both PSQL error codes we could have them in the same error handling middleware, and then if we wanted to separate that out into a new PSQL Error Handling function.